x/observer/types/ballot.go

```diff
@@ -90,28 +94,35 @@ func CreateVotes(listSize int) []VoteType {
 // BuildRewardsDistribution builds the rewards distribution map for the ballot
 // It returns the total rewards units which account for the observer block rewards
 func (m Ballot) BuildRewardsDistribution(rewardsMap map[string]int64) int64 {
+	// If the ballot is in progress, return 0, we do not want to distribute rewards for in progress ballots
```

Comment:
The current function should replicate the older logic exactly, without any changes.  Please keep that in mind when you look it over. 
The older logic did not explicitly define a Condition for `BallotStatus_BallotInProgress` but behaved the same way as none of the cases matched.